apps/condo/domains/organization/schema/Organization.js

```diff
@@ -78,6 +80,26 @@ const Organization = new GQLListSchema('Organization', {
             type: Json,
             isRequired: false,
         },
+        phone: {
+            schemaDoc: 'Normalized organization phone in E.164 format without spaces',
+            type: Text,
+            hooks: {
+                resolveInput: async ({ resolvedData }) => {
+                    const newValue = normalizePhone(resolvedData['phone'], true)
+                    return newValue || resolvedData['phone']
+                },
+                validateInput: async ({ resolvedData, addFieldValidationError }) => {
+                    const newValue = normalizePhone(resolvedData['phone'], true)
+                    if (resolvedData['phone'] && newValue !== resolvedData['phone']) {
+                        addFieldValidationError(`${PHONE_WRONG_FORMAT_ERROR}phone] invalid format`)
+                    }
+                },
+            },
+        },
+        phoneNumberPrefix: {
```

Comment:
what's the prefix? Is like `+7`, `+1` or smth like this? Should it be validated or so?

Reply:
It's numeric identifier assigned to a specific line in software for, used when outgoing call before the number to be called. For example `phoneNumberPrefix` = 01, then the result phone number to be called = 01`phone number`.

Update `schemaDoc` for this field.

**Upd:** PR for backend located here https://github.com/open-condo-software/condo/pull/3406
Here it was an accident because of the bad rebase